app/services/api/v1x2.rb

```diff
@@ -0,0 +1,55 @@
+require 'services/api/v1x1'
```

Comment:
Should the `require` in `app/controllers/api/v1x2.rb` update to use this file? https://github.com/RedHatInsights/catalog-api/pull/767/files#diff-a1c2402e0294f62a6907e3005462e0deR1

Reply:
Yes, you're right, it should 👍